st2common/bin/st2ctl

```diff
@@ -116,6 +116,11 @@ function service_manager() {
 function reopen_component_log_files() {
   COM=${1}
 
+  if [[ "$COM" == "mistral" ]]; then
+    echo "Sorry, Mistral does not yet support gracefully reopening log files"
+    return 1
+  fi
+
   PID=`ps ax | grep -v grep | grep -v st2ctl | grep "${COM}" | awk '{print $1}'`
```

Comment:
If we enhance `PID` expression with `grep -E` later, this line would be good to sync with that version as well.

Reply:
Good point. I'll make a similar change here